lib/index.js

```diff
@@ -24,8 +23,7 @@ let submit = (options) => {
   return new Promise((resolve, reject) => {
     database(options)
       .then(client => {
-        fs
-          .createReadStream(options.metrics)
+        process.stdin
```

Comment:
Closing the loop, no. `cli.stdin` reads input as JSON, while this reads in line-delimited JSON.